src/jit-format/jit-format.cs

```diff
@@ -104,7 +104,8 @@ private void SetPlatform()
                         }
                         else
                         {
-                            _os = match.Groups[1].Value.Trim();
+                            // Assuming anything other than Windows or OSX is a Linux flavor
+                            _os = "Linux";
```

Comment:
Why not match "Linux" or whatever for this case and then error gracefully in another else clause?  We already know xxxBSDs are out there.